src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -279,6 +279,12 @@ protected override void Dispose(bool disposing)
                 _readBio?.Dispose();
                 _writeBio?.Dispose();
             }
+
+            if(_alpnHandle.IsAllocated)
```

Comment:
Style: `if<space>(`

Reply:
Fixed and a couple of other space issues (new line after an if statement etc)